### PR TITLE
T20 timed run functionality

### PR DESCRIPTION
**Pull request by jacobktm**
Added an automated timed test run for t20 testing to 9801-logger.jl

### CODE REVIEW

File: 9801-logger/9801-logger.jl [hunk 1]
```diff
@@ -169,149 +196,170 @@ try
...
-    volts_bounds = []
-    freqs        = []
-    reals        = []
-    apps         = []
-    pfacs        = []
-    currs        = []
-    apks         = []
+    
+    test_count   = 1
+    while (test_count <= TEST_RUNS)
```
Comment (13r0ck) [inline 9801-logger/9801-logger.jl]: There is no need to have `test_count` and `test_runs` variables. just use a for loop on `tests`

**Review by 13r0ck — CHANGES_REQUESTED** [review]

File: 9801-logger/9801-logger.jl [hunk 2]
```diff
@@ -169,149 +194,169 @@ try
...
-    volts_bounds = []
-    freqs        = []
-    reals        = []
-    apps         = []
-    pfacs        = []
-    currs        = []
-    apks         = []
+    
+    for test in TESTS
+      println("")
```
Comment (13r0ck) [inline 9801-logger/9801-logger.jl:203]: why is this here?

Comment (jacobktm) [reply 9801-logger/9801-logger.jl:203]: formatting, so all the lines aren't jumbled together

Comment (13r0ck) [reply 9801-logger/9801-logger.jl:203]: I'd move 
```julia
      if (AUTOMATED)
        println("\nRunning $(test) test for ~$(RUN_TIME) seconds")
      end
```
up a few lines and just add the `\n` to the print statement

Comment (jacobktm) [reply 9801-logger/9801-logger.jl:203]: that would make the informative purpose of that line confusing since it would put that statement above the potential sleep statement when it actually executes in the opposite order

Comment (13r0ck) [reply 9801-logger/9801-logger.jl:203]: https://github.com/system76/system76-ee/pull/10/commits/e19d498e9c7a805e7c09a780ee68388b54a98696 is in my opinion worse. There are better ways to do it, but I think it's fine for now to just revert it to the way it was.

Comment (jacobktm) [reply 9801-logger/9801-logger.jl:203]: alrighty, I've reverted it

**Review by 13r0ck — APPROVED** [review]
